src/Objects/Update.php

```diff
@@ -82,47 +95,21 @@ public function detectType(): ?string
             'poll',
             'poll_answer',
         ];
-
-        return $this->collect()
-            ->keys()
-            ->intersect($types)
-            ->pop();
     }
 
     /**
      * Get the message contained in the Update.
+     *
+     * @return Message|EditedMessage|InlineQuery|ChosenInlineResult|CallbackQuery|ShippingQuery|PreCheckoutQuery|Poll|PollAnswer
      */
     public function getMessage()
     {
-        switch ($this->detectType()) {
-            case 'message':
-                return $this->message;
-            case 'edited_message':
-                return $this->edited_message;
-            case 'channel_post':
-                return $this->channel_post;
-            case 'edited_channel_post':
-                return $this->edited_channel_post;
-            case 'inline_query':
-                return $this->inline_query;
-            case 'chosen_inline_result':
-                return $this->chosen_inline_result;
-            case 'callback_query':
-                if (isset($this->callback_query->message)) {
-                    return $this->callback_query->message;
-                }
-                break;
-            case 'shipping_query':
-                return $this->shipping_query;
-            case 'pre_checkout_query':
-                return $this->pre_checkout_query;
-            case 'poll':
-                return $this->poll;
-            case 'poll_answer':
-                return $this->poll_answer;
-        }
+        return $this->{$this->detectType()};
+    }
 
-        return $this;
+    public function getEvent()
+    {
+        return 'Telegram\\Bot\\Events\\' . Str::ucfirst(Str::camel($this->detectType())) . "Event";
```

Comment:
Please change to `Str::studly()` method here.

Reply:
Just use this:

```php
'Telegram\Bot\Events\\' . Str::studly($this->detectType()) . 'Event'
```

Changed:
- Change to `studly` method.
- No need to escape slashes for every word.
- Single quote for `Event` at the end to make it consistent.

Reply:
Force push done. All this was changed.